examples/case7.jl

```diff
@@ -208,7 +206,6 @@ function get_sub_system_data(a_id, products, T)
                 SingleInvData(
                     FixedProfile(1e7 / cap), # Capex [NOK/MW]
                     FixedProfile(cap),       # Max installed capacity [MW]
-                    0,                       # initial capacity [MW]
```

Comment:
Not that this is no longer necessarily equivalent to the previous application. That would have been by changing it to `FixedProfile(0)`. Then again, it does not really matter.

Reply:
Right, so since the initial capacity in this case is zero, the mathematical formulation is equivalent right?

Reply:
Yes, that is right. I did not check the part which was not shown.